src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [].filter((animal) => animal.health > 0);
```

Comment:
The static alive array is being filtered upon declaration, which will not work as intended since it's supposed to be an empty array at this point. Instead, the filtering should be done when necessary, such as after an animal's health changes.